README.md

```diff
@@ -1,3 +1,81 @@
 # loopback-cli
 
 LoopBack CLI tool for creating projects, models and more.
+This package supersedes the older `slc` tool.
+
+## Installation
+
+```
+$ npm install loopback-cli
+```
+
+## Usage
+
+### Getting started
+
+ 1. Run `lb` to create a new LoopBack application
```

Comment:
IMO, we should really use lb to show usage/help menu like most command line apps. I think it's weird we don't follow typical conventions. This is a good chance to make it right. `lb create|app` should create a new app. Might be considered out of scope/feature request though.